### PR TITLE
Pylinting

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: pyscal/factory.py [hunk 1]
```diff
@@ -553,7 +553,11 @@ def create_scal_recommendation(
             raise ValueError('"base" case not supplied')
         if "high" not in params:
             raise ValueError('"high" case not supplied')
-        if not all([isinstance(x, dict) for x in params.values()]):
+        if not (
+            isinstance(params["low"], dict)
+            and isinstance(params["base"], dict)
+            and isinstance(params["high"], dict)
+        ):
```
Comment (asnyv) [inline pyscal/factory.py:560]: Did `pylint` really prefer this over the more compact old one, or is it enough to remove the `[ ]` as you have done later in similar cases?

**Review by asnyv — APPROVED** [review]
A couple of comments, but looks ok 😊

File: tests/test_factory.py [hunk 2]
```diff
@@ -887,6 +887,14 @@ def test_scalrecommendation():
...
         pyscal_factory.create_scal_recommendation(baselow)
 
+    with pytest.raises(
+        ValueError, match="All values in parameter dict must be dictionaries"
+    ):
+
+        pyscal_factory.create_scal_recommendation(
+            {"low": [1, 2], "base": {"swl": 0.1}, "high": {"swl": 0.1}}
+        )
+
```
Comment (asnyv) [inline tests/test_factory.py:897]: Maybe belongs in a different PR, but I think it's ok to add a test

Comment (berland) [reply tests/test_factory.py:897]: borderline case, I didn't dare to fix the pylint issue without testing it..

File: pyscal/factory.py (hunk 1 above)
Comment (berland) [reply pyscal/factory.py:560]: pylint didn't like the list generator, that could have been fixed by removing square brackets, but then I found the explicit way of writing it more readable.
